cluster/log-dump/logexporter-daemonset.yaml

```diff
@@ -67,6 +67,8 @@ spec:
           requests:
             cpu: 10m
             memory: 10Mi
+      nodeSelector:
+        topology.kubernetes.io/zone: {{.Zone}}
```

Comment:
Could we get that conditionally? Only for regional clusters?

Reply:
OK, I'm fine with leaving empty nodeSelector:

Reply:
We will get that conditionally after using `LOGEXPORTER_ZONE`, as suggested by yourself ;). Resolving the remark.